builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -653,12 +704,17 @@ func resourceVSphereVirtualMachineCreate(d *schema.ResourceData, meta interface{
 							vm.bootableVmdk = v
 						}
 					} else {
-						return fmt.Errorf("template, size, or vmdk argument is required")
+						return fmt.Errorf("clone, size, or vmdk argument is required")
 					}
 				}
 				if v, ok := disk["datastore"].(string); ok && v != "" {
 					vm.datastore = v
 				}
+				log.Printf("[DEBUG] vm.clone: %#v\n", vm.clone)
+
+				if deprecatedLinkedClone, ok := d.GetOk("linked_clone"); ok && deprecatedLinkedClone.(bool) && vm.clone != nil {
```

Comment:
It is automatically logged to the user because it's marked as a deprecated field in the schema.